File: JAVA/UberPool.java
public class UberPool extends Car{
    // Propiedades propias de la clase UberX
    String brand;
    String model;

    //Constructor que coincide con la clase padre, heredado de la clase car
    public UberPool(String license, Account driver, String brand, String model){
        super(license, driver); 
        this.brand = brand;
        this.model = model;
    }
    
}

File: PHP/uberPool.php
<?php
require_once('car.php');
class uberPool extends Car{
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        //Acceso a la super clase
        parent::__construct($license, $driver);
        $this->brand = $brand;
        $this->model = $model;
    }
}

?>